components/layout/components/nav-bar.js

```diff
@@ -1,128 +1,139 @@
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://gitlab.com/gableroux/unity3d-gitlab-ci-example"
-            >
-              Example CI project
-            </a>
-          </Item>
-        </ItemGroup>
-        <ItemGroup title="Docker">
-          <Item icon={<GitlabOutlined />} key="external:gableroux:unity3d">
-            <a target="_blank" rel="noreferrer" href="https://hub.docker.com/r/gableroux/unity3d">
-              gableroux/unity3d
-            </a>
-          </Item>
-        </ItemGroup>
-        <ItemGroup title="GitHub Actions">
-          <Item icon={<GithubOutlined />} key="external:action:overview">
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://github.com/webbertakken/unity-actions"
-            >
-              Unity Actions
-            </a>
-          </Item>
-          <Item icon={<GithubOutlined />} key="external:action:builder">
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://github.com/webbertakken/unity-builder"
-            >
-              Unity Builder
-            </a>
-          </Item>
-          <Item icon={<GithubOutlined />} key="external:action:test-runner">
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://github.com/webbertakken/unity-test-runner"
-            >
-              Unity Test Runner
-            </a>
-          </Item>
-          <Item icon={<GithubOutlined />} key="external:action:activate">
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://github.com/webbertakken/unity-activate"
-            >
-              Unity Activate
-            </a>
-          </Item>
-          <Item icon={<GithubOutlined />} key="external:action:request-activation-file">
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://github.com/webbertakken/unity-request-activation-file"
-            >
-              Unity Request Activation
-            </a>
-          </Item>
-          <Item icon={<GithubOutlined />} key="external:action:return-license">
-            <a
-              target="_blank"
-              rel="noreferrer"
-              href="https://github.com/webbertakken/unity-return-license"
-            >
-              Unity Return License
-            </a>
-          </Item>
-        </ItemGroup>
-      </SubMenu>
-      <Item>
-        <Link href="/discord">
-          <DiscordLogo height={30} style={{ verticalAlign: '-0.75em' }} />
-        </Link>
-      </Item>
-    </Menu>
-  </Header>
-);
+const NavBar = () => {
+  const { asPath: currentKey } = useRouter();
+  let defaultSelectedKeys = '';
+  if (currentKey === '/') {
+    defaultSelectedKeys = '/';
+  } else if (currentKey.includes('/docs')) {
+    defaultSelectedKeys = '/docs';
+  }
```

Comment:
Please note that my pr also rebuilds almost everything in this component, due to search becoming part of it in #73.